test-cypress/integration/explore/visual.graph.spec.js

```diff
@@ -382,393 +228,352 @@ describe('Visual graph screen validation', function () {
+        // Include inferred: false
+        getIncludeInferredStatementsCheckbox().check()
+            .should('be.checked');
+        // Expand results over owl:sameAs: false
+        getSameAsCheckbox().check()
+            .should('be.checked');
+        // Show predicate labels: true
+        getShowPredicateLabelsCheckbox().uncheck()
+            .should('not.be.checked');
+
+        // No pre-added preferred/ignored types
+        getPreferredTypesField().type('vin:PinotNoir')
+            .should('have.value', 'vin:PinotNoir');
+        getShowPreferredTypesOnlyCheckbox().check()
+            .should('be.checked');
+        getIgnoredTypesField().type('vin:PinotNoir')
+            .should('have.value', 'vin:PinotNoir');
+
+        // Go to predicates tab
+        openPredicatesTab();
+        // No pre-added preferred/ignored predicates
+        getPreferredPredicatesField().type('vin:hasSugar')
+            .should('have.value', 'vin:hasSugar');
+        getShowPreferredPredicatesOnlyCheckbox().check()
+            .should('be.checked');
+        getIgnoredPredicatesField().type('vin:hasSugar')
+            .should('have.value', 'vin:hasSugar');
+
+        saveSettings();
+
+        // Reset settings and verify everything is reverted to its default
+        openVisualGraphSettings();
+        resetSettings();
+        saveSettings();
+
+        openVisualGraphSettings();
+        // Verify that the default settings are as follows:
+        // Maximum links to show: 20
+        getLinksNumberField().should('have.value', '20');
+        // Preferred lang: en
+        cy.get('.preferred-languages .tag-item').should('have.length', 0);
+        // Include inferred: false
+        getIncludeInferredStatementsCheckbox().should('not.be.checked')
+            .and('not.be.disabled');
+        // Expand results over owl:sameAs: false
+        getSameAsCheckbox().should('not.be.checked')
+            .and('be.disabled');
+        // Show predicate labels: true
+        getShowPredicateLabelsCheckbox().should('be.checked')
+            .and('not.be.disabled');
+
+        // No pre-added preferred/ignored types
+        getPreferredTypesField().should('be.empty');
+        getShowPreferredTypesOnlyCheckbox().should(('not.be.checked'))
+            .and('not.be.disabled');
+        getIgnoredTypesField().should('be.empty');
+
+        // Go to predicates tab
+        openPredicatesTab();
+        // No pre-added preferred/ignored predicates
+        getPreferredPredicatesField().should('be.empty');
+        getShowPreferredPredicatesOnlyCheckbox().should(('not.be.checked'))
+            .and('not.be.disabled');
+        getIgnoredPredicatesField().should('be.empty');
     });
+
+    // Visual graph home view access
+
+    function getSearchField() {
+        return cy.get('.search-rdf-resources input:visible');
+    }
+
+    function typeInSearchField(resource) {
+        getSearchField().type(resource).trigger('change').wait(200).type('{enter}');
```

Comment:
Document the wait

Reply:
will try

Reply:
I removed it.

Reply:
I reverted this and even increased it a bit because after the removal the tests started to fail much more. I put some description of course.